src/query/v2/requests.hpp

```diff
@@ -387,7 +387,8 @@ struct GetPropertiesResponse {
 enum class EdgeDirection : uint8_t { OUT = 1, IN = 2, BOTH = 3 };
 
 struct ExpandOneRequest {
-  // TODO(antaljanosbenjamin): Filtering based on the id of the other end of the edge?
+  // To get Edges with with destination vertex edge_wist_st
+  std::optional<VertexId> edge_with_dst;
```

Comment:
I think it suffices for now as we only expanding one tupple at a time. We can change this when we support batched expansions

Reply:
My concern here is the rest of the request and the shard rsm is capable of receiving multiple vertices to expand from. The limitation is only present in the query engine, but we have tests that uses the requests and shard rsm directly, thus this change introduce inconsistency in terms of `ExpandOneRequest`. I think the change is quite small (vector instead of optional plus the changes on how it is accessed), but makes the request consistent.